cmd/podman/play/kube.go

```diff
@@ -117,7 +117,7 @@ func init() {
 		_ = kubeCmd.RegisterFlagCompletionFunc(configmapFlagName, completion.AutocompleteDefault)
 
 		buildFlagName := "build"
-		flags.BoolVar(&kubeOptions.Build, buildFlagName, false, "Build all images in a YAML (given Containerfiles exist)")
+		flags.BoolVar(&kubeOptions.Build, buildFlagName, true, "Build all images in a YAML (given Containerfiles exist)")
```

Comment:
`Containerfiles` or `a Containerfile`?

Reply:
@TomSweeneyRedHat Yes cause yaml could container multiple images as well. I think that's why it was `Containerfiles` from before.